README.md

```
@@ -210,7 +210,7 @@ If you believe a skill in this list should be flagged or has a security concern,
 | [Search & Research](#search--research) (345) | [iOS & macOS Development](#ios--macos-development) (29) | [Security & Passwords](#security--passwords) (54) |
 | [Clawdbot Tools](#clawdbot-tools) (37) | [Transportation](#transportation) (111) | [Moltbook](#moltbook) (29) |
 | [CLI Utilities](#cli-utilities) (180) | [Personal Development](#personal-development) (52) | [Gaming](#gaming) (35) |
-| [Health & Fitness](#health--fitness) (87) | | |
+| [Health & Fitness](#health--fitness) (89) | | |
```

markdownlint-cli2 (0.23.0)

[warning] Link fragments should be valid

(MD051, link-fragments)